inflection/src/inflection/dictionary/DictionaryKeyIterator.cpp

```diff
@@ -18,24 +18,22 @@ DictionaryKeyIterator::DictionaryKeyIterator(const metadata::MarisaTrieIterator<
 }
 
 DictionaryKeyIterator::DictionaryKeyIterator(DictionaryKeyIterator&& iterator)
-    : trieIterator(iterator.trieIterator)
+    : trieIterator(iterator.trieIterator.release())
```

Comment:
If you're removing the npc checks elsewhere, then this line needs an null pointer check added here.